src/battle/battle_anim_813F0F4.c

```diff
@@ -86,6 +81,284 @@ static void sub_8141CBC(struct Sprite *sprite);
 static void sub_8141CF4(struct Sprite *sprite);
 static void sub_8141D20(struct Sprite *sprite);
 
+extern const u8 gBattleAnimSpriteSheet_Particles[];
+const struct CompressedSpriteSheet gBallOpenParticleSpritesheets[] =
+{
+    {gBattleAnimSpriteSheet_Particles, 0x100, 55020},
```

Comment:
should I make the last arg here a hex value? I think I meant to do that earlier

Reply:
yes, tags should be hex